svd/rp2040.yaml

```
@@ -633,6 +633,16 @@ IO_BANK0:
       "GPIO*_STATUS": {}
       "GPIO*_CTRL": {}
 
+SPI0:
+  SSPCR0:
+    _modify:
+      FRF:
+        description: "Frame format."
+    FRF:
+      _replace_enum:
+        Motorola: [0, "Motorola SPI frame format"]
+        TexasInstrument: [1, "TI synchronous serial frame format"]
+        NationalMicrowire: [2, "National Microwire frame format"]
```

```suggestion
        Motorola: [0, "Motorola SPI frame format"]
        TexasInstruments: [1, "Texas Instruments synchronous serial frame format"]
        NationalSemiconductorMicrowire: [2, "National Semiconductor Microwire frame format"]
```
(The third variant's name gets quite long this way, but omitting half of the company's name seems wrong.)

---

I already found `NationalMicrowire` in all caps barely readable, so I'd like to avoid `NATIONALSEMICONDUCTORMICROWIRE`.  
Would `TI` and `NSM` be ok?

---

Ouch, it doesn't separate the words with underscores when converting to all-caps?

What about using underscores in the first place? The original svd doesn't use camel case for other enums either.

`MOTOROLA`, `TEXAS_INSTRUMENTS`, `NATIONAL_SEMICONDUCTOR_MICROWIRE` don't look too unreadable.